AFQ/viz/utils.py

```diff
@@ -27,36 +27,48 @@
-                          "FP": tableau_20_rgb[8],
-                          "FA": tableau_20_rgb[9],
-                          "IFO_L": tableau_20_rgb[10],
-                          "IFO_R": tableau_20_rgb[11],
-                          "ILF_L": tableau_20_rgb[12],
-                          "ILF_R": tableau_20_rgb[13],
-                          "SLF_L": tableau_20_rgb[14],
-                          "SLF_R": tableau_20_rgb[15],
-                          "UNC_L": tableau_20_rgb[16],
-                          "UNC_R": tableau_20_rgb[17],
-                          "ARC_L": tableau_20_rgb[18],
-                          "ARC_R": tableau_20_rgb[19]})
-
-POSITIONS = OrderedDict({"ATR_L": (1, 0), "ATR_R": (1, 4),
-                         "CST_L": (1, 1), "CST_R": (1, 3),
-                         "CGC_L": (3, 1), "CGC_R": (3, 3),
-                         "FP": (4, 2), "FA": (0, 2),
-                         "IFO_L": (4, 1), "IFO_R": (4, 3),
-                         "ILF_L": (3, 0), "ILF_R": (3, 4),
-                         "SLF_L": (2, 1), "SLF_R": (2, 3),
-                         "ARC_L": (2, 0), "ARC_R": (2, 4),
-                         "UNC_L": (0, 1), "UNC_R": (0, 3)})
+COLOR_DICT = OrderedDict({
+    "ATR_L": tableau_20_rgb[0], "C_L": tableau_20_rgb[0],
+    "ATR_R": tableau_20_rgb[1], "C_R": tableau_20_rgb[1],
+    "CST_L": tableau_20_rgb[2],
+    "CST_R": tableau_20_rgb[3],
+    "CGC_L": tableau_20_rgb[4], "MCP": tableau_20_rgb[4],
+    "CGC_R": tableau_20_rgb[5], "CCMid": tableau_20_rgb[5],
+    "HCC_L": tableau_20_rgb[6],
+    "HCC_R": tableau_20_rgb[7],
+    "FP": tableau_20_rgb[8], "CC_ForcepsMinor": tableau_20_rgb[8],
+    "FA": tableau_20_rgb[9], "CC_ForcepsMajor": tableau_20_rgb[9],
+    "IFO_L": tableau_20_rgb[10], "IFOF_L": tableau_20_rgb[10],
+    "IFO_R": tableau_20_rgb[11], "IFOF_R": tableau_20_rgb[11],
+    "ILF_L": tableau_20_rgb[12], "F_L": tableau_20_rgb[12],
+    "ILF_R": tableau_20_rgb[13], "F_R": tableau_20_rgb[13],
+    "SLF_L": tableau_20_rgb[14],
+    "SLF_R": tableau_20_rgb[15],
+    "UNC_L": tableau_20_rgb[16], "UF_L": tableau_20_rgb[16],
+    "UNC_R": tableau_20_rgb[17], "UF_R": tableau_20_rgb[17],
+    "ARC_L": tableau_20_rgb[18], "AF_L": tableau_20_rgb[18],
+    "ARC_R": tableau_20_rgb[19], "AF_R": tableau_20_rgb[19]})
+
+POSITIONS = OrderedDict({
+    "ATR_L": (1, 0), "ATR_R": (1, 4), "C_L": (1, 0), "C_R": (1, 4),
+    "CST_L": (1, 1), "CST_R": (1, 3),
+    "CGC_L": (3, 1), "CGC_R": (3, 3), "MCP": (3, 1), "CCMid": (3, 3),
+    "FP": (4, 2), "FA": (0, 2),
+    "CC_ForcepsMinor": (4, 2), "CC_ForcepsMajor": (0, 2),
+    "IFO_L": (4, 1), "IFO_R": (4, 3), "IFOF_L": (4, 1), "IFOF_R": (4, 3),
+    "HCC_L": (4, 0), "HCC_R": (4, 4),
+    "ILF_L": (3, 0), "ILF_R": (3, 4), "F_L": (3, 0), "F_R": (3, 4),
+    "SLF_L": (2, 1), "SLF_R": (2, 3),
+    "ARC_L": (2, 0), "ARC_R": (2, 4), "AF_L": (2, 0), "AF_R": (2, 4),
+    "UNC_L": (0, 1), "UNC_R": (0, 3), "UF_L": (0, 1), "UF_R": (0, 3)})
+
+BUNDLE_RECO_2_AFQ = \
```

Comment:
This one might be more useful in `AFQ.data`, as we might want it for comparisons that don't have anything to do with visualization.

Reply:
Done! In the long run we may want to further break up both files. See #427